app/courses/[courseId]/scoreboard/page.tsx

```diff
@@ -0,0 +1,160 @@
+import TableCell from "@mui/material/TableCell";
+import TableContainer from "@mui/material/TableContainer";
+import TableHead from "@mui/material/TableHead";
+import TablePagination from "@mui/material/TablePagination";
+import TableRow from "@mui/material/TableRow";
+import { useParams, useRouter, useSearchParams } from "next/navigation";
+import { graphql, useLazyLoadQuery } from "react-relay";
+import { pageScoreboardQuery } from "@/__generated__/pageScoreboardQuery.graphql";
+import { Button } from "@mui/material";
+import Link from "next/link";
+
+interface Column {
+  id: "place" | "name" | "power";
+  label: string;
+  minWidth?: number;
+  align?: "right" | "center";
+}
+
+const columns: readonly Column[] = [
+  { id: "place", label: "Place", minWidth: 100 },
+  { id: "name", label: "Name", minWidth: 200, align: "center" },
+  {
+    id: "power",
+    label: "Power",
+    minWidth: 100,
+    align: "right",
+  },
+];
+
+interface Data {
+  place: string;
+  name: string;
+  power: number;
+}
+
+function createData(place: string, name: string, power: number): Data {
+  return { place, name, power };
+}
+
+function capitalizeFirstLetter(str: string): string {
+  return str.charAt(0).toUpperCase() + str.slice(1);
+}
+
+export default function StickyHeadTable() {
+  const [page, setPage] = React.useState(0);
+  const [rowsPerPage, setRowsPerPage] = React.useState(10);
+
+  const params = useParams();
+  const courseId = params.courseId;
+
+  const { scoreboard } = useLazyLoadQuery<pageScoreboardQuery>(
+    graphql`
+      query pageScoreboardQuery($courseId: UUID!) {
+        scoreboard(courseId: $courseId) {
+          user {
+            userName
+          }
+          powerScore
+        }
+      }
+    `,
+    { courseId: courseId }
+  );
+
+  const handleChangePage = (event: unknown, newPage: number) => {
+    setPage(newPage);
+  };
+
+  const handleChangeRowsPerPage = (
+    event: React.ChangeEvent<HTMLInputElement>
+  ) => {
+    setRowsPerPage(+event.target.value);
+    setPage(0);
+  };
+
+  const rows: any[] = [];
+
+  scoreboard.forEach((element, index: number) => {
+    index++;
+    rows.push(
+      createData(
+        `${index}`,
+        capitalizeFirstLetter(element.user.userName),
+        element.powerScore
+      )
+    );
+  });
+
+  return (
+    <div>
+      <Link href={`/courses/${courseId}`}>
+        <Button variant="contained" sx={{ margin: "10px" }}>
+          Back
+        </Button>
+      </Link>
+      <Paper sx={{ width: "100%", overflow: "hidden" }}>
+        <TableContainer sx={{ maxHeight: "100%" }}>
+          <Table stickyHeader aria-label="sticky table">
+            <TableHead>
+              <TableRow>
+                {columns.map((column) => (
+                  <TableCell
+                    key={column.id}
+                    align={column.align}
+                    style={{ minWidth: column.minWidth }}
+                    sx={{ backgroundColor: "#90a4ae" }}
```

Comment:
ich würde überlegen ob wir dein custom-styling der tabelle hier und weiter unten rausnehmen, finde das ungestylte von material schicker um ehrlich zu sein:

<img width="1459" alt="image" src="https://github.com/IT-REX-Platform/gits-frontend/assets/18666999/fbb3eeec-4f25-430b-8c41-169e59404daf">

Reply:
evtl sogar das "Paper" drum rum weglassen?

<img width="1482" alt="image" src="https://github.com/IT-REX-Platform/gits-frontend/assets/18666999/f1c6c042-5e79-45e9-bc96-17fd0ad7cdba">

Reply:
Fun fact: Ich habe von der MUI Seite einfach den Code für "Sticky Header" tables kopiert. Ist also das MUI styling. 
Aber ja, kann ich machen